src/_base/css/font-awesome-4.0.3.css

```diff
@@ -1336,3 +1336,10 @@
 .fa-plus-square-o:before {
   content: "\f196";
 }
+.lilv2 {
+    padding-left:24px;
+}
+.lilv3 {
+    padding-left:48px;
+}
```

Comment:
These changes should go in `base.css` (and you'll want to put a space after the colon).